elasticsearch/sgconfig/config.yml

```diff
@@ -4,7 +4,7 @@ opendistro_security:
       xff:
         enabled: true
         remoteIpHeader: 'x-forwarded-for'
-        trustedProxies: '.*'
+        trustedProxies: '0\:0\:0\:0\:0\:0\:0\:1|127\.0\.0\.1'
```

Comment:
Are you sure this works also for address like `::1`? In addition, IPv6 provides also link-local address, e.g. ~`fe::80...`. Should these be also trusted?